### DISCUSSION TITLE
When will you support pie chart? Any plan?

### DISCUSSION THREAD

**Discussion by wuzhefang** [ChartsCSS/charts.css#66] [General]
I found it in tag 0.9.0

**Comment by ramiy** [comment]
Hi @wuzhefang,

I guess that when browsers will support [CSS math functions](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions#math_functions) I will be able to support the circular charts.

Meanwhile you can use [conic gradients](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/conic-gradient()) to create pie charts on a single HTML element.
